common/config.go

```diff
@@ -22,28 +22,49 @@ const (
 var homeDir, _ = os.UserHomeDir()
 var baseDir = path.Join(homeDir, ".atomicswap")
 
+// MoneroNode represents the host and port of monerod's RPC endpoint
+type MoneroNode struct {
+	Host string
+	Port uint
+}
+
 // Config contains constants that are defaults for various environments
 type Config struct {
-	DataDir          string
-	MoneroDaemonHost string
-	MoneroDaemonPort uint
-	ContractAddress  ethcommon.Address
-	Bootnodes        []string
+	DataDir         string
+	MoneroNodes     []*MoneroNode
+	ContractAddress ethcommon.Address
+	Bootnodes       []string
 }
 
 // MainnetConfig is the mainnet ethereum and monero configuration
 var MainnetConfig = Config{
-	DataDir:          path.Join(baseDir, "mainnet"),
-	MoneroDaemonHost: "127.0.0.1",
-	MoneroDaemonPort: DefaultMoneroDaemonMainnetPort,
+	DataDir: path.Join(baseDir, "mainnet"),
+	MoneroNodes: []*MoneroNode{
+		{
+			Host: "127.0.0.1",
+			Port: DefaultMoneroDaemonMainnetPort,
```

Comment:
Obviously, we want to encourage people to use their own nodes, but I could put some default nodes in here.